sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -0,0 +1,96 @@
+## API Report File for "@azure/schema-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export interface SchemaId {
+    // (undocumented)
+    id: string;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClient extends SchemaRegistryClientContext {
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+    getIdBySchemaContent(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetIdBySchemaContentResponse>;
+    getSchemaById(schemaId: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetSchemaByIdResponse>;
+    registerSchema(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientRegisterSchemaResponse>;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClientContext extends coreHttp.ServiceClient {
+    // (undocumented)
+    $host: string;
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+}
+
+// @public
+export interface SchemaRegistryClientGetIdBySchemaContentHeaders {
+    // (undocumented)
+    location?: string;
+    xSchemaId?: string;
+    xSchemaIdLocation?: string;
+    xSchemaVersion?: number;
+    xSerialization?: string;
+}
+
+// @public
+export type SchemaRegistryClientGetIdBySchemaContentResponse = SchemaRegistryClientGetIdBySchemaContentHeaders & SchemaId & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: SchemaId;
+        parsedHeaders: SchemaRegistryClientGetIdBySchemaContentHeaders;
+    };
+};
+
+// @public
+export interface SchemaRegistryClientGetSchemaByIdHeaders {
+    // (undocumented)
+    location?: string;
+    xSchemaId?: string;
+    xSchemaIdLocation?: string;
+    xSchemaVersion?: number;
+    xSerialization?: string;
+}
+
+// @public
+export type SchemaRegistryClientGetSchemaByIdResponse = SchemaRegistryClientGetSchemaByIdHeaders & {
+    body: string;
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: string;
+        parsedHeaders: SchemaRegistryClientGetSchemaByIdHeaders;
+    };
+};
+
+// @public
+export interface SchemaRegistryClientOptionalParams extends coreHttp.ServiceClientOptions {
+    endpoint?: string;
+}
+
+// @public
+export interface SchemaRegistryClientRegisterSchemaHeaders {
```

Comment:
this interface shows up a lot under different names - should be just one interface I think?

Reply:
Good feedback to send back to the swagger PR

Reply:
cc @arerlend

Reply:
@jhendrixMSFT is there a way to create definitions for header groups in swagger 2?  I believe it is in v3.

Reply:
(If it's not possible with v2, it won't be blocking as we can ensure no duplication like this in what we actually export.)

Reply:
Sadly no, this is something not in v2.